### ISSUE TITLE
Cannot log into Passbolt extension anymore - Firefox 73.0.1 + Passbolt 2.12.1 extension.

### ISSUE THREAD

**Issue by ValentinFlorea** [passbolt/passbolt_browser_extension#94]
## ISSUE NAME Cannot log into Passbolt extension anymore - Firefox 73.0.1 + Passbolt 2.12.1 extension.
* Passbolt Version: Passbolt_API: 3eaf5fcf6457b477cd5371b33f8f1a80e53e9273 (2.12)
* Platform and Target:
-- Browser: Firefox 73.0.1 on Mac OS X 10.15.3
-- Web server: Apache/2.4.18 (Ubuntu) 
-- etc.: Firefox Passbolt extension version 2.12.1

### What you did
This morning I couldn't log into Passbolt anymore. Noticed that the passbolt extension was updated 2 days ago (and yesterday I didn't try using it). So most likely the latest update broke something.

### What happened
Passbolt webpage: After filling in the correct password on the passbolt webpage, a spinner appears shortly, then no error is shown, instead the login form appears again. 
Passbolt extension small window: a spinner appears after filling in the password, then it disappears and it looks like I'm logged in, but instead another small spinner appears next to "Retrieving your passwords". This second spinner doesn't go away.

Things I tried:
- incognito mode
- disabling all other extensions + incognito
- reinstalling the extension
- using the extension on Chrome - THIS WORKED (on chrome I have Passbolt extension 2.12.0 installed). So this is proof that it isn't a problem with my passbolt account.
- A colleague used on Firefox an older version of the extension (2.11.0) and it also worked for him. 

### What you expected to happen
To be able to log into passbolt.

**Comment by cedricalfonsi** [comment]
Hello @ValentinFlorea,

Thank you for the report. 

I'm not able to reproduce the issue on the exact same environment as you.

Could you provide us with:
- The passbolt browser extension log. You can access it as following:
-- Go to about:debugging#/runtime/this-firefox
-- Click on the inspect link in the passbolt extension section

Did you try to shutdown completely the browser (on MacOS closing it put it in idle) ?

**Comment by ValentinFlorea** [comment]
Thank you a lot for the quick response, Cedric. Figured it out with your help. It was indeed specific to my setup. 

I activated block 3rd party cookies in my Firefox a few days ago.
After accessing the about:debugging tab and investigating the console log, saw that it was complaining about not being able to store cookies. 

**Important**: after I relaxed the 3rd party cookies block setting, it didn't work immediately. In the console it would complain about the same 3rd party cookie block. I had to reinstall the extension, but then that fixed it.

Thank you a lot for your help 👍 
This can be closed.
